wolfcrypt/src/integer.c

```diff
@@ -330,7 +330,7 @@ int mp_copy (mp_int * a, mp_int * b)
   }
 
   /* grow dest */
-  if (b->alloc < a->used) {
+  if (b->alloc < a->used || b->dp == NULL) {
```

Comment:
Didn't we talk about 0 / NULL case in original changes.  Why grow if not actually needed?  If no copy needed is there a better way to suppress warning?